vignettes/metadata.Rmd

```diff
@@ -0,0 +1,102 @@
+---
+title: "Adding metadata"
+author: "Thierry Onkelinx"
+output:
+  rmarkdown::html_vignette:
+        fig_caption: yes
+vignette: >
+  %\VignetteIndexEntry{Adding metadata}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Introduction
+
+`git2rdata` supports extra metadata since version 0.4.1.
+Metadata is stored in a separate file with the same name as the data file, but with the extension `.yml`.
+The metadata file is a YAML file with a specific structure.
+The metadata file contains a generic section and a section for each variable in the data file.
+The generic section contains information about the data file as a whole.
+The variable sections contain information about the variables in the data file.
+The metadata file is stored in the same directory as the data file.
+
+The generic section contains the following mandatory fields, automatically created by `git2rdata`:
```

Comment:
Similarly your use of `fields` would need to map to something else. Not entirely sure, I think, it's called `property` in frictionless.